src/main/java/mate/academy/spring/dto/request/MovieSessionRequestDto.java

```diff
@@ -1,10 +1,15 @@
 package mate.academy.spring.dto.request;
 
 import java.time.LocalDateTime;
+import javax.validation.constraints.Min;
+import javax.validation.constraints.NotNull;
 
 public class MovieSessionRequestDto {
+    @Min(1)
     private Long movieId;
+    @Min(1)
     private Long cinemaHallId;
```

Comment:
what about id = null ? of for some reason id start with 0
I think its better to use @NotNull here